From 2c76719007bc70705cc7cc72ecf10aaf57ecc7fd Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Mon, 7 Jan 2013 18:22:28 +0100
Subject: [PORT FROM MAIN] wifi: modify reply length for DRIVER cmd

BZ: 75654

Broadcom driver does not support a so big buffer.
It is needed to reduce it for DRIVER command.

Change-Id: I4b536bee976a36fbbe22b1aeff63c81fc28b974c
Orig-Change-Id: If5ae06fe3313ac8153cec18ea97e900f0c557625
Category: device enablement
Domain: CWS.WIFI-Bluewoods
Origin: upstream
Upstream-Candidate: no,proprietary
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
---
 wpa_supplicant/ctrl_iface.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wpa_supplicant/ctrl_iface.c b/wpa_supplicant/ctrl_iface.c
index 39bbc8c..913f4b7 100644
--- a/wpa_supplicant/ctrl_iface.c
+++ b/wpa_supplicant/ctrl_iface.c
@@ -5765,7 +5765,7 @@ char * wpa_supplicant_ctrl_iface_process(struct wpa_supplicant *wpa_s,
 #ifdef ANDROID
 	} else if (os_strncmp(buf, "DRIVER ", 7) == 0) {
 		reply_len = wpa_supplicant_driver_cmd(wpa_s, buf + 7, reply,
-						      reply_size);
+						      4096);
 #endif
 	} else if (os_strcmp(buf, "REAUTHENTICATE") == 0) {
 		pmksa_cache_clear_current(wpa_s->wpa);
-- 
1.7.4.1

